### PR TITLE
add in macOS cloud info

### PR DESCRIPTION
**Pull request by sebastian-lerner**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by sebastian-lerner** [comment]
@thomasr8 or @rosieyohannan when you get a chance can you take a look please? Not 100% sure if I did the formatting correctly with sub-bullets

**Review by thomasr8 — APPROVED** [review]

**Comment by sebastian-lerner** [comment]
thanks @thomasr8  any chance you can merge when you get a chance? i don't have write access =/

File: jekyll/_cci2/ip-ranges.md
Comment (rosieyohannan) [inline jekyll/_cci2/ip-ranges.md:135]:
```suggestion
The machines that execute *all jobs* on CircleCI’s platform, not just jobs opted into IP ranges, are hosted on Amazon Web Services (AWS), Google Cloud Platform (GCP), and CircleCI's macOS Cloud. An exhaustive list of IP addresses that CircleCI’s traffic may come from on these cloud providers’ platforms can be found by looking up each cloud provider's IP address ranges. AWS & GCP offer endpoints to find this information.
```

**Comment by rosieyohannan** [comment]
Thanks @sebastian-lerner! I just made one tiny change and will merge when the build passes ☺️

**Pull request merged**
